_includes/pbc/all/install-features/202212.0/install-the-dynamic-multiple-stores.md

```diff
@@ -2518,6 +2518,95 @@ Also you need to change domain name for all endpoints in EU region.
 Please, check `deploy.dev.dynamic-store.yml` file for more details.
 
 
+## Data migration 
+
+You don't need to migrate data in the database for the dynamic store feature. Only changes in the search and storage engine are necessary.
+Propel migration will be executed automatically and added new tables for dynamic store feature and populate new tables with data via data import.
+For correct work of dynamic store feature you need to add new data structure for search and storage engines.
+
+
+First of all, you need to truncate all data from search and storage engines. 
+Please see below for more details in the section **"Truncate data from search and storage engines"**.
+Next step is run sync:data command for populate new data structure for search and storage engines.
+
+```bash
+vendor/bin/console sync:data
+```
+
+All data will be populated in search and storage engines.
+
+{% info_block warningBox "Verification" %}
+
+
+### Truncate data from search and storage engines
+
+#### Storage engine
+
+Since Dynamic Store feature uses Redis as a key-value store for each region instead of store. If using same Redis instance for region, you need to clear all data from Redis for existing stores.
```

Comment:
```suggestion
The Dynamic Store feature uses Redis as a key-value store for each region instead of store. If you are using the same Redis instance for a region, you need to clear all data from Redis for existing stores.
```